js/helpers/uploadProfilePicHelper.js

```diff
@@ -0,0 +1,19 @@
+import { firebaseDB } from "../config/firebaseConfig";
+import { AsyncStorage } from "react-native";
+
+export const updateProfilePic = async profilePhoto => {
+  try {
+    const uid = await AsyncStorage.getItem("user");
+    await firebaseDB
+      .collection("users")
+      .doc(uid)
+      .set(
+        {
+          profilePhoto
+        },
+        { merge: true }
+      );
+  } catch (err) {
+    console.log(err);
```

Comment:
Console logging is not ideal. What would be better is managing error state here instead so the UI could be updated to give the user feedback that the write was not successful.

Because you have chosen to not use Redux, this gets a bit messier (with Redux you could have just dispatched an action here to update the state). Without Redux, you may need to resort to adding a callback function as a parameter to call when the right fails (where you could call this.setState() where this helper is used in your component.

Similarly, if the write is successful above do you need to update state in your application in response to this? Again, without Redux and async action creators, another callback param may be needed here so you can call this.setState(), if needed.